submit.py

```diff
@@ -68,11 +69,13 @@ def write_general_header(self, f, mem=3000, walltime_hours=14,
             outdir = os.path.join(os.getcwd(),'out')
             if not os.path.isdir(outdir):
                 os.mkdir(outdir)
-            self.write_line(f, "#PBS -o %s/${PBS_JOBID}.out"%outdir)
-            self.write_line(f, "#PBS -e %s/${PBS_JOBID}.err"%outdir)
+            self.write_line(f, "#PBS -o %s/${PBS_JOBID}.out" % outdir)
+            self.write_line(f, "#PBS -e %s/${PBS_JOBID}.err" % outdir)
```

Comment:
These two files above would be overwritten for each job in an array since the job id is the same, wouldn't they? In that case, using `${PBS_ARRAY_INDEX}` might be an option (if PBS behaves the same as LSF, this index is always 0 for a regular job).